tests/installation/encrypted_volume/activate_encrypted_volume.pm

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test module to activate encrypted volume.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my $encrypted_volume = $testapi::distri->get_encrypted_volume();
+    $encrypted_volume->enter_volume_encryption_password($password);
```

Comment:
this is fine at the level of the page that we enter something and we accept it but from the point of view of the test should look more like one single operation: `$testapi::distri->get_encrypted_volume_activation()->activate($password)`